apps/cli/cleanup_stale_in_progress.py

```
@@ -0,0 +1,356 @@
+    try:
+        reports_dir = os.environ.get("REPORTS_DIR", "reports")
+        if summary.get("dry_run"):
+            path = os.path.join(
+                reports_dir, "D1", "d1_drift_preview.jsonl",
+            )
+        else:
+            path = os.path.join(reports_dir, "D1", "d1_drift.jsonl")
+        os.makedirs(os.path.dirname(path) or ".", exist_ok=True)
+        with open(path, "a", encoding="utf-8") as f:
+            f.write(json.dumps(summary, ensure_ascii=False) + "\n")
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)